app/assets/javascripts/modules/FirmMap.js

```diff
@@ -14,7 +14,9 @@ define(['jquery', 'DoughBaseComponent'],
   var FirmMapProto,
       defaultConfig = {
         zoomLevel: 11,
-        center: {lat: 0, lng: 0}
+        center: {lat: 0, lng: 0},
+        adviserPinUrl: '/assets/pins/adviser.svg',
```

Comment:
Is it worth checking for svg support before using these, and then have an image fallback?

You can use:

```js
if ($('body').hasClass('svg')) {
  // use svg
}
```